src/lib/pickles_types/plonk_types.ml

```diff
@@ -855,6 +964,15 @@ module Evals = struct
       ; opt uses_lookup
       ; Typ.array ~length:5 (opt lookup_sorted) (* TODO: Fixme *)
       ; opt feature_flags.runtime_tables
+      ; opt feature_flags.runtime_tables
+      ; opt feature_flags.xor
+      ; opt feature_flags.lookup
+      ; opt
```

Comment:
Can we add a small comment saying what it is representing. As said in a previous PR, it is for our future us to avoid going back in the code and loosing time.

Reply:
Why do we add rot?